pandas/tests/tseries/holiday/test_holiday.py

```diff
@@ -187,7 +191,9 @@ def test_holidays_within_dates(holiday, start, expected):
     # holiday was not in the original date range (e.g., 7/4/2015 -> 7/3/2015).
     if isinstance(holiday, str):
         calendar = get_calendar("USFederalHolidayCalendar")
+        name = holiday
         holiday = calendar.rule_from_name(holiday)
+        assert holiday is not None, name
```

Comment:
what is the purpose here?

Reply:
If the tests request a holiday by an old/incorrect name, I wanted the test to fail with an indication of which holiday name was at issue.

Reply:
this is not useful to do as the test would fail anyhow. if you are not testing something then we don't want code for it.

Reply:
Gotcha.  Removed.